CHANGELOG.md

```diff
@@ -32,6 +42,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 [#3185]: https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/3185
 [v3.12.0]: https://github.com/SumoLogic/sumologic-kubernetes-collection/releases/v3.12.0
 
+## [v3.11.1]
+
+### Released 2023-08-10
+
+### Fixed
+
+- fix: disable zstd compression internally [#3197]
+
+[#3197]: https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/3197
+[v3.11.1]: https://github.com/SumoLogic/sumologic-kubernetes-collection/releases/v3.11.1
+
```

Comment:
> We also drop the changelog entry for https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/3197, as it's already in 3.12.1.
Do we? I think I don't understand this comment

Reply:
I deleted the respective changelog entry file in `.changelog`, so we don't get this again in the 3.13 changelog.